### DISCUSSION TITLE
File Upload To Have Circle Option

### DISCUSSION THREAD

**Discussion by jason-beach** [filamentphp/filament#8045] [Ideas]
I have a client that is requesting to have a cropper that would be able to not only do squares, rectangles, free hand, etc. but to also have a circle cropper and also oval. Would be nice to have the file upload, that already does many things, to also have the ability to do circle and oval cropping.

**Comment by leandrocfe** [comment]
v2? https://github.com/Nuhel/FilamentCroppie

**Comment by jason-beach** [comment]
I'm on V3...

I'm attempting to create a PR with this specific feature. So far I was able to achieve a circle cropper but I'm finalizing code before I submit a PR.
